bug_report_page.php

```diff
@@ -624,7 +624,7 @@
 ?>
 	<tr>
 		<th class="category">
-			<label for="file"><?php echo lang_get( $t_file_upload_max_num == 1 ? 'upload_file' : 'upload_files' ) ?></label>
+			<label for="ufile[]"><?php echo lang_get( $t_file_upload_max_num == 1 ? 'upload_file' : 'upload_files' ) ?></label>
```

Comment:
I suggest we just use plural.  The old config option default (of 1) was based on the old UI of adding more than one browse control, but in the new UI, limiting max attachments to 1 doesn't make sense.  I suggest a default value of 10.  To limit a user to upload no more than 10 attachments at a time.